.github/workflows/build.yml

```diff
@@ -193,7 +193,7 @@ jobs:
           python-version: '3.8'
     - name: Install Requirements
       run: |  # Custom pyinstaller built with https://github.com/yt-dlp/pyinstaller-builds
-          python -m pip install --upgrade pip setuptools wheel "py2exe<0.12"
+          python -m pip install --upgrade pip setuptools wheel "py2exe<0.13"
```

Comment:
is pinning the version nesecery

Reply:
Py2exe increases that number every time there's a breaking change. I imagine that's why @SG5 decided to pin it.

That said, it is a bit of a double edged sword, because I highly doubt anyone here will notice when Py2exe `0.13` gets released, and then yt-dlp will be using an outdated version of py2exe

Reply:
@gamer191 is right.
`py2exe` releases breaking changes in minor versions. Eventually we would have an outdated version of the library or broken build of `yt-dlp`

Reply:
I pinned it only because of the current API change. I would rather not pin it after this PR. It's easier for me to fix and re-update when it breaks than to keep track of py2exe updates

Reply:
Ok, let me remove version pinning